docs(title): component playgrounds (#2607)
ionic-team/ionic-docs: docs/api/title.md

<EncapsulationPill type="shadow" />

> `ion-title` is a component that sets the title of the `Toolbar`.

Reviewer: ```suggestion
Title is a text component that sets the title for a [toolbar](./toolbar). It can be used to describe the screen or section a user is currently on or the app being used.
```

Feel free to change the content here, just trying to fluff it a bit.

## Basic

import Basic from '@site/static/usage/title/basic/index.md';

<Basic />

## Collapsible Large Titles

> The Large Title will display when the `ion-content` is scrolled to the start of the scroll container. When the title is scrolled behind the header, the condensed title will fade in.

Reviewer: ```suggestion
The large title will display when the [content](./content) is scrolled to the start of the scroll container. When the title is scrolled behind the header, the condensed title will fade in.
```

:::info  

This feature is only available for iOS.

:::

import CollapsibleLargeTitle from '@site/static/usage/title/collapsible-large-title/basic/index.md';

<CollapsibleLargeTitle />

### Collapsible Buttons

> The [`ion-buttons`](./buttons.md) component can be used with the [`collapse`](./buttons.md#collapse) property to additionally display in the header as the toolbar is collapsed.

Reviewer: ```suggestion
The [buttons](./buttons.md) component can be used with the [`collapse`](./buttons.md#collapse) property to additionally display in the header as the toolbar is collapsed.
```

import CollapsibleLargeTitleButtons from '@site/static/usage/title/collapsible-large-title/buttons/index.md';

<CollapsibleLargeTitleButtons />

## Theming

> The collapsible large title should appear seamless in relation to the rest of your content. This means that the background color of the `ion-toolbar` that contains the collapsible large title should always match the background color of `ion-content`.

Reviewer: ```suggestion
The collapsible large title should appear seamless in relation to the rest of your content. This means that the background color of the toolbar containing the collapsible large title should always match the background color of the content.
```

> By default, the `ion-toolbar` that contains the standard title is hidden using `opacity: 0` and is progressively shown as you collapse the large title by scrolling. As a result, the background color that you see behind the standard title is actually the background color of `ion-content`.

Reviewer: ```suggestion
By default, the toolbar that contains the standard title is hidden using `opacity: 0` and is progressively shown as you collapse the large title by scrolling. As a result, the background color that you see behind the standard title is actually the background color of content.
```

> You can change the background color of the toolbar with the standard title by setting the `--background` CSS variable on `ion-toolbar`. This will give the effect of the header changing color as you collapse the large title.

Reviewer: ```suggestion
You can change the background color of the toolbar with the standard title by setting the `--background` CSS variable. This will give the effect of the header changing color as you collapse the large title.
```
